co-simulation/bundle/src/assembly/resources/scenarios/Town04/sumo/Town04.sumocfg

```diff
@@ -5,13 +5,12 @@
     </traci_server>
     <input>
         <net-file value="Town04.net.xml"/>
-        <route-files value="Town04.rou.xml"/>
-        <additional-files value="detector.xml"/>
-        <gui-settings-file value="gui.settings.xml"/>
-
+        <route-files value="carlavtypes.rou.xml,Town04.rou.xml"/>
+        <additional-files value="detector.xml, bus_stop.add.xml"/>
     </input>
     <gui_only>
         <tracker-interval value="0.1"/>
+        <gui-settings-file value="viewsettings.xml"/>
```

Comment:
Why did this change?

Reply:
This is for a different purpose, this is for representing vehicle types in the sumo gui